worldengine/simulations/irrigation.py

```diff
@@ -5,7 +5,7 @@ class IrrigationSimulation(object):
     def is_applicable(world):
         return world.has_watermap() and (not world.has_irrigation())
 
-    def execute(self, world, seed):#seed is currently not used
+    def execute(self, world, seed):
```

Comment:
it is fine, execute is part of an interface, in the other implementations seed is used